ambry-frontend/src/main/java/com/github/ambry/frontend/s3/S3MultipartCompleteUploadHandler.java

```diff
@@ -258,7 +258,7 @@ private Callback<String> idConverterCallback(BlobInfo blobInfo, String blobId) {
         String serviceId = blobInfo.getBlobProperties().getServiceId();
         retryExecutor.runWithRetries(retryPolicy,
             callback -> router.updateBlobTtl(blobId, serviceId, Utils.Infinite_Time, callback,
-                QuotaUtils.buildQuotaChargeCallback(restRequest, quotaManager, false)), this::isRetriable,
+                QuotaUtils.buildQuotaChargeCallback(restRequest, quotaManager, false), null, null), this::isRetriable,
```

Comment:
Just a note. I think we may have to move id conversion inside Router here too later (when we enable dual writes for Multipart uploads).

Reply:
Yeah, we need to remove id convert logic in all router APIs later.